hydrus/data/crud.py

```diff
@@ -254,11 +209,8 @@ def insert_multiple(objects_: List[Dict[str,
 
```

Comment:
Can we not reuse insert here as is? What is preventing us to use insert in insert_mutliple? All these checks are already in insert, maybe we should reuse that function?

Reply:
Yes, makes sense. I will make this change